### ISSUE TITLE
Error during connect to provider: could not allocate provider IP NET: no more unused subnets

### ISSUE THREAD

**Issue by n10ty** [mysteriumnetwork/node#5517]
```
failed to connect: failed to start service: server responded with an error: 500 (http://localhost:4050/connection) [err_connect] Failed to connect: could not send p2p session create request: peer error: cannot start session: : cannot get provider config for session 9797395c-1ae2-4197-8fae-0fd6d4b80664: could not create provider mode wg config: could not allocate provider IP NET: no more unused subnets
```

Not a single bad provider. Example:
0x85a46fbdc80fefe643f8fc39c098f58cc2fbf6a6
0xd6dddbb3e093d2aac3198aff79c3d6909e7da9dc
0x74e7c179e4dd37e8d6545166d00e3cd905192825

Docker image, Mysterium Node
  Version: 1.17.4
  Build info: Branch: 1.17.4. Build id: 648396996. Commit: 10ae5a99
But it's reproduces on any node version

**Comment by lollapalooza1973** [comment]
Same here, on version 1.18.1
I believe this is related to issue [5508](https://github.com/mysteriumnetwork/node/issues/5508)
Please check if you experience same behavior.

**Comment by soffokl** [comment]
I hope this will get resolved by 1.18.3, many providers have already updated.

**Comment by Nitrooo** [comment]
> I hope this will get resolved by 1.18.3, many providers have already updated.

Is there a way to know what version a provider is using? Or maybe, even better, general statistics about the whole network?

**Comment by soffokl** [comment]
For general statistics about the whole network we have https://stats.mysterium.network/
Unfortunately, it does not share information about the version, only general things.
